src/games/gamemodelitem.py

```diff
@@ -15,13 +16,34 @@ def __init__(self, game, me, preview_dler):
 
         self.game = game
         self.game.gameUpdated.connect(self._game_updated)
+        self.hosted_at = None  # put here as long it's not coming from server
```

Comment:
Unclear why you're proposing to add code for a feature that definitely won't work. Maybe add that part after filing the corresponding server PR?